### ISSUE TITLE
Where did CodeSnap save the screenshots in MacOS?

### ISSUE THREAD

**Issue by zainsadaqat** [kufii/CodeSnap#92]
## CodeSnap Snapshots are awesome

I saw many developers using CodeSnap and I really liked it but when I installed this extension and took a screenshot of my code I found no way to find that screenshot. There's no proper instruction on where my screenshot is going. It's not in my screenshot folder too.

Can someone please tell me about this?

_Thanks in Advance!_

**Comment by duckimann** [comment]
It should show you a dialog to choose the place to save the screenshot OR copy to the clipboard if you using Ctrl + C

**Comment by zainsadaqat** [comment]
No, It doesn't show me any dialogue box and Ctrl + C, Ctrl + V doesn't work
too.

**Comment by Lanceqt** [comment]
I have the same issue no dialogue appears

**Comment by Aworldc** [comment]
Assuming it does, when the "📸CodeSnap" pane opens, did you click the CodeSnap logo that appears above the preview? if you didn't, you need to click that to save.
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/83331712/161402662-99e3dc82-ba76-4dae-9278-bc18c36b4ca6.png"> 
hopefully this helps.

**Comment by Aworldc** [comment]
found that in [the readme](https://github.com/kufii/CodeSnap/blob/master/README.md#usage-instructions) #usage-instructions

**Comment by ldexterldesign** [comment]
👋 https://github.com/kufii/CodeSnap/issues/126
